pyproject.toml

```diff
@@ -0,0 +1,78 @@
+# pyproject.toml
```

Comment:
I upgraded from `setup.py` to the newer `pyproject.toml` packaging configuration, so that the CI tests with hatchling would be easy to do.
The new config file should have all the same information as the old one.